pkgs/development/libraries/wlroots/protocols.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitLab, pkg-config, wayland-scanner }:
+
+stdenv.mkDerivation rec {
+  pname = "wlr-protocols";
+  rev = "d998ee6fc64ea7e066014023653d1271b7702c09";
+  version = "unstable-2021-11-01";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "wlroots";
+    repo = "wlr-protocols";
+    rev = "${rev}";
+    sha256 = "1vw8b10d1pwsj6f4sr3imvwsy55d3435sp068sj4hdszkxc6axsr";
+  };
+
+  depsBuildBuild = [ pkg-config ];
```

Comment:
this is only needed needed in depsBuildBuild when meson is used otherwise it only goes in nativeBuildInputs